qutebrowser/browser/hints.py

```diff
@@ -748,6 +760,13 @@ def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
         self._context.tab.find_all_elements(selector, self._start_cb,
                                             only_visible=True)
 
+    def current_mode(self):
+        """ Returns the currently active hinting mode (or None otherwise). """
+        if not self._context or not self._context.hint_mode:
+            return None
+
+        return self._context.hint_mode
```

Comment:
Good suggestion. Is the lack of a newline between the `if` body and the `return` a python stylistic thing? Looks weird to me

> It only gets called by HintKeyParser (i.e., in hint mode), so self._context should always be set, no?

I don't see a need for this function to make that assumption. Who knows what might use it in the future?